RxKeyboard.podspec

```diff
@@ -11,8 +11,8 @@ Pod::Spec.new do |s|
   s.frameworks       = 'UIKit', 'Foundation'
   s.requires_arc     = true
 
-  s.dependency 'RxSwift', '>= 4.1.2'
-  s.dependency 'RxCocoa', '>= 4.1.2'
+  s.dependency 'RxSwift', '>= 4.0.0'
```

Comment:
I think this should be `RxSwift`, '~> 4` or it won't catch 4.1.

Reply:
Get it! 

I thought the signal >= would take greater than or equal to 4.0.0

Reply:
I might be wrong actually. Not on my computer, will check later

Reply:
No problem @freak4pc. I think the sinal ~> work as well

Reply:
Ok, so the existing code should be fine